client/components/tinymce/plugins/simple-payments/style.scss

```diff
@@ -103,6 +103,7 @@
 		position: relative;
 		margin-bottom: 20px;
 		margin-right: 25px;
+		width: 200px;
```

Comment:
Nice catch @Copons! Regression needlessly caused by https://github.com/Automattic/wp-calypso/pull/24806/files#diff-c6fa3312c33f32b46e0ff478d51ee5f2L103. Could we also restore `height: 200px` here just in case.